zipline/assets/assets.py

```diff
@@ -91,6 +95,35 @@
 })
 
 SymbolOwnership = namedtuple('SymbolOwnership', 'start end sid symbol')
+OwnershipPeriod = namedtuple('OwnershipPeriod', 'start end sid value')
```

Comment:
Do you think we need both of these, or could we use `OwnershipPeriod` for symbols too?